### PR TITLE
Mona Quickstart

### PR DESCRIPTION
**Pull request by HungryWizard**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/mlops/mona/config.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - mona
+  - machine-learning
+  - machine learning
+  - monitoring
+  - data monitoring
+  - data
+  - integrity
+  - data integrity
+  - drift
+  - explainability
```
Comment (jpvajda) [inline quickstarts/mlops/mona/config.yml:44]:
```suggestion
```

File: quickstarts/mlops/mona/config.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - ai
+  - mona
+  - machine-learning
+  - machine learning
+  - monitoring
+  - data monitoring
+  - data
+  - integrity
+  - data integrity
+  - drift
```
Comment (jpvajda) [inline quickstarts/mlops/mona/config.yml:43]:
```suggestion
```

File: quickstarts/mlops/mona/config.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+keywords:
+  - mlops
+  - ml
+  - ai
+  - mona
+  - machine-learning
+  - machine learning
+  - monitoring
+  - data monitoring
+  - data
```
Comment (jpvajda) [inline quickstarts/mlops/mona/config.yml:40]:
```suggestion
```

File: quickstarts/mlops/mona/config.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - Mona Labs
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - mlops
+  - ml
```
Comment (jpvajda) [inline quickstarts/mlops/mona/config.yml:33]:
```suggestion
```

File: quickstarts/mlops/mona/config.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - monalabs-mlops
+
+documentation:
+  - name: Mona Labs Docs
+    # TODO(elie): Update with the correct link.
+    url: https://docs.newrelic.com/docs/integrations/mlops-integrations/algorithmia-mlops-integration/
```
Comment (jpvajda) [inline quickstarts/mlops/mona/config.yml:55]: Is mona and alogorthmia the same thing?

Comment (jpvajda) [reply quickstarts/mlops/mona/config.yml:55]: please use the proper YAML formatting >- for documentation URL references.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation

File: install/mlops/mona/install.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+id: monalabs-mlops
+name: Mona Labs
+
+# TODO(elie): Consider changing.
+title: Mona Labs Intelligent Monitoring for AI.
+
+description: |
+  Mona provides the best visibility into AI systems in order to reduce associated risks with production AI, optimize operational processes, and enable teams to plan better AI roadmaps. The Mona - New Relic integration provides users with the ability to automatically view Mona generated insights directly on their New Relic dashboard, explore production AI / ML data with NRQL, and connect specific Mona insight events to New Relic Incident Intelligence.
```
Comment (jpvajda) [inline install/mlops/mona/install.yml:8]: @bradleycamacho any thoughts on the description or summary? looks good to me but I thought I'd have you also check it out.

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**